plugins/orchestrator-backend/src/service/ScaffolderService.ts

```diff
@@ -91,22 +90,14 @@ export class ScaffolderService {
       workspacePath: workspacePath,
       logger: this.logger,
       logStream: this.streamLogger,
-      createTemporaryDirectory: async () => {
-        const tmpDir = await fs.mkdtemp(`${workspacePath}_step-${0}-`);
-        tmpDirs.push(tmpDir);
```

Comment:
we can review later, but I think at some point we need to cleanup the temp dirs that were created to execute the actions, in theory, they should be removed once the workflow is completed but it might be a long-running workflow, that is the trick part.